app/build.gradle

```diff
@@ -51,8 +51,8 @@ dependencies {
     implementation "androidx.compose.ui:ui:$compose_version"
     implementation "androidx.compose.material:material:$compose_version"
     implementation "androidx.compose.ui:ui-tooling-preview:$compose_version"
-    implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.3.1'
-    implementation 'androidx.activity:activity-compose:1.3.1'
+    implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.4.1'
```

Comment:
Updated the androidx.lifecycle dependencies to ensure project would properly function. 
https://stackoverflow.com/questions/28683327/how-to-check-if-gradle-dependency-has-new-version 
https://kotlinlang.org/docs/coding-conventions.html#names-for-backing-properties

Reply:
Careful when updating library versions.  Some libraries may depend on others, so when you upgrade one, you may have dependencies on others.